TFNetwork.py

```diff
@@ -1447,11 +1461,11 @@ def _tf_summary(self):
       return  # skip summaries. the root net should also do this
     name = self.get_tf_name()
     if self._loss_value is not None:
-      tf.summary.scalar("loss_%s" % name, self._loss_value * self._norm_factor)
+      tf.summary.tensor_summary("loss_%s" % name, self._loss_value * self._norm_factor)
```

Comment:
Why this change?

Reply:
Ah, because it is not necessarily a scalar now?
Add a comment (extra line, above this) about that.

Reply:
we actually are not sure, it did crash before. (even without positional loss)

Reply:
Do not change code when you are not sure. Understand what the problem is. But I guess it's what I wrote.

Reply:
Also, when you understood what the problem is, add a comment into the code, why you use `tensor_summary` here, and not `scalar`.